csstools/index.html

```diff
@@ -1,786 +1,392 @@
+        </section>
+
+        <section>
+          <h2>Browser Bugs</h2>
+          <ul>
+            <li>Not all browsers render CSS the same, with the most number of issues in IE.</li>
+            <li>The best defense is to be aware and to test.</li>
+          </ul>
+          <ul>
+            <li><a href="http://coding.smashingmagazine.com/2010/06/07/the-principles-of-cross-browser-css-coding/">The Principles of Cross-Browser CSS Coding</a></li>
+            <li><a href="http://www.tutorialfeed.org/2010/01/10-common-css-browser-compatibility.html">10 Common CSS Browser Compatibility Issues</a></li>
+            <li><a href="http://archive.webstandards.org/css/winie/">IE's Top 10 CSS Problems</a></li>
+          </ul>
+        </section>
+
+        <section>
+          <h2>Browser Testing</h2>
+          <p>Since not all browsers work on all operating systems (IE on Mac), you may need help from these tools for cross-browser testing:
+          </p>
+          <ul>
+            <li><a href="http://www.modern.ie/en-us/virtualization-tools">Modern.ie</a> Microsoft's free access to testing tools</li>
+            <li><a href="http://www.saucelabs.com">SauceLabs</a>: Cloud-based automated and manual testing</li>
+            <li><a href="http://www.browserstack.com/">BrowserStack</a>: Lets you debug in virtual browsers.</li>
+            <li><a href="http://browsershots.org/">BrowserShots</a>: Takes screenshots of your sites in different browsers.</li>
+            <li><a href="https://browserling.com/">BrowserLing</a>: Ditto.</li>
+            <li><a href="http://multipleies.en.softonic.com/">MultipleIEs</a>: Makes it possible to test in multiple IE versions on one computer.</li>
+            <li><a href="http://www.my-debugbar.com/wiki/IETester/HomePage">IETester</a>: Ditto.</li>
+          </ul>
+        </section>
+
+        <section>
+         <h2>Cross-Device CSS</h2>
+        </section>
+
+        <section>
+          <h2>CSS Media Queries</h2>
+          <p>A media query consists of a media type and an optional combination of media features. CSS rules inside the query will be applied
+            only if the query is true.</p>
+            <pre>
+              @media &lt;media types&gt; and &lt;media features&gt; {
+              /* media-specific rules */
+            }
+          </pre>
+          <p>They can also be specified for a linked stylesheet:</p>
+          <pre>
+            &lt;link rel="stylesheet" media="&lt;media types&gt;
+            and &lt;media features&gt;" href="alternative.css" /&gt;
+          </pre>
+          <p class="notes">
+            https://developer.mozilla.org/en/CSS/@media#Media_groups
+            https://developer.mozilla.org/en/CSS/Media_queries
+          </p>
+        </section>
+
+        <section>
+          <h2>Print Styles</h2>
+          <p>All modern browsers support the "print" media type for specifying print styles.</p>
+          <pre>
+            @media print {
```

Comment:
here's an example of how this segment can go:
```
<pre><code>@media print {
  body {
    background: white;
  }
  #header, #footer {
    display: none;
  }
  .section {
    page-break-inside: avoid;
    page-break-after: always;
  }
}
</code> </pre>
```

## Before
<img width="1045" alt="screen shot 2016-07-30 at 1 18 58 pm" src="https://cloud.githubusercontent.com/assets/2753545/17272913/76b07f4a-5658-11e6-99a3-2e7d858904e4.png">

## After
<img width="948" alt="screen shot 2016-07-30 at 1 20 18 pm" src="https://cloud.githubusercontent.com/assets/2753545/17272914/7d0a2954-5658-11e6-815c-f3fdffd49009.png">